<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentedObjectViewer</class>
 <widget class="QMainWindow" name="SegmentedObjectViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimoxCGAL - SegmentedObject Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>440</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Visualization</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxColModel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>460</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Collision Model</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonLoadSegmented">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxSegments">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>680</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelSegments">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>650</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Segment</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelOut">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>710</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Segments:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonScreenshot">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Screenshot</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
